### DISCUSSION TITLE
Do the Manage Learn services code fall directly under the SunbirdEd Building Block, and thereby in its release cycle.

### DISCUSSION THREAD

**Discussion by aks30** [Sunbird-Ed/Community#32] [General Discussion]
Do the code changes that we have to make for any bug fixes or new capability related to the Manage Learn use cases, which have to be done in Manage Learn services will be part of the SunbirdEd Building Block directly?

Service Repo - 

[Survey](https://github.com/project-sunbird/ml-survey-service)
[Manage Learn Core](https://github.com/project-sunbird/ml-core-service)
[Projects](https://github.com/project-sunbird/ml-projects-service)
[Manage Learn Reports](https://github.com/project-sunbird/ml-reports-service)

Cc - @vijiurs, @kiranharidas187 @aishwaryashikshalokam, @Prateek-slokam

**Comment by luckynvdu** [comment]
All these repos are part of Sunbird ED only but if you think frequent changes to these repos are not possible in intermediate release (once in 15 days) then you can plan your changes for major release only (once in 12 weeks).
We still need to have an explicit conversation internally for the same.

**Reply by aks30** [reply]
Sure thank you @luckynvdu , this helps
